src/server/scripts/Northrend/zone_borean_tundra.cpp

```diff
@@ -1751,18 +1757,45 @@ class npc_bonker_togglevolt : public CreatureScript
                 player->FailQuest(QUEST_GET_ME_OUTA_HERE);
         }
 
-        void UpdateEscortAI(uint32 /*diff*/) override
+        void EnterCombat(Unit* who) override
         {
-            if (GetAttack() && UpdateVictim())
+            if (who->GetTypeId() != TYPEID_PLAYER)
+                if (roll_chance_i(20))
+                    Talk(SAY_BONKER_5);
+        }
+
+        void UpdateAI(uint32 diff) override
```

Comment:
why the change from ```UpdateEscortAI``` to ```UpdateAI``` ?

Reply:
NPC must say an out of combat text every once in a while.

I'm not sure if UpdateEscortAI executes if the escort hasn't started yet. Is there any real difference between the two?

Reply:
https://github.com/TrinityCore/TrinityCore/blob/3.3.5/src/server/game/AI/ScriptedAI/ScriptedEscortAI.cpp#L187

Reply:
Updated, thanks.

Reply:
my pleasure, keep up the good work